src/components/select/select-particles.js

```diff
@@ -7,7 +7,10 @@ import { gmi } from "../../core/gmi/gmi.js";
 
 const onPointerOut = (emitter, button) => button.on(Phaser.Input.Events.POINTER_OUT, () => emitter.stop());
 
+const isButtonEnabled = button => button.listeners(Phaser.Input.Events.POINTER_UP).length === 0;
+
 const createEmitter = (scene, button, config) => {
+    if (isButtonEnabled(button)) return;
```

Comment:
You can access button.config I think. 

Maybe...

but this works.

Reply:
hmm it wasn't on there, is that part of the state? and is the state config set on the button config at all?